docs: update AppKit terminology to Reown across documentation (#738)
reown-com/reown-docs: appkit/unity/core/usage.mdx

title: Usage
---

After you have installed and configured Reown,
you can start using it to interact with the wallet provider and blockchain.

## EVM

> Reown comes with the [Nethereum](https://nethereum.com/) and [Wagmi](https://wagmi.sh/) integration out of the box.

Reviewer: ```suggestion
The Reown SDK offers [Nethereum](https://nethereum.com/) and [Wagmi](https://wagmi.sh/) integration out of the box.
```

#### Quick start

> Initialize Reown once on startup, then use the extension methods on `Web3`.

Reviewer: ```suggestion
Initialize the SDK once on startup, then use the extension methods on `Web3`.
```

}
```

For full setup (prefab, logging, Android/iOS/WebGL notes), see the [Reown docs](/appkit/unity/core/installation).

#### Using the extension methods

The adapter adds extension methods to `Web3`.

> ##### Try to resume a Reown session

Reviewer: ```suggestion
##### Try to resume a session
```

Once you call `LoginAppKit` (or a session resume succeeds), the adapter sets `web3.WalletBase = AppKitWalletBase`. From that point:

- Existing `Web3` calls keep working.

> - Any required signatures are requested through Reown.

Reviewer: ```suggestion
- Any required signatures are requested through Reown AppKit.
```
